hack/release.toml

```diff
@@ -107,6 +107,17 @@ Talos starting with 1.10 will have ISO's that will use GRUB only for legacy BIOS
         description = """\
 Talos now provides `ethtool`-style Ethernet low-level configuration via `network/EthernetConfig` documents.
 Current status of the interface can be read by `talosctl get ethernetstatus`.
+"""
+
+    [notes.toolchain]
+        title = "Fully bootstrapped builds"
+        description = """\
+Talos 1.10 is built with a toolchain based on [\[Stageˣ\]](https://stagex.tools/), which is a project building fully bootstrapped software.
```

Comment:
unsure about square brackets and how are they escaped when generating changelogs

Reply:
do `make release-notes` and you'll see :)